src/Tools/Auditor/EntryPoint.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using Bootstrap.Tools.Auditor;
+
+namespace Microsoft.Dafny.Compilers.SelfHosting.Auditor;
+
+public class Auditor : Plugins.Rewriter {
+
+  private readonly DafnyAuditor auditor = new();
+
+  public Auditor(ErrorReporter reporter) : base(reporter) { }
+
+  public override void PostResolve(Program program) {
+    var text = auditor.Audit(program);
+    // TODO: write to the console or file depending on options
+    Console.WriteLine(text);
```

Comment:
Let's log a request on the tracker to have command line plugins?
Also, it would be nice to be able to exit with an error if certain problems are present.  Perhaps the user could specify tags to error about?

Reply:
https://github.com/dafny-lang/dafny/issues/2735

Reply:
I think I'd like to add the ability to exit with an error as part of a separate change.

Reply:
Let's create an issue for this

Reply:
#22